tests/catalog/catalogItems.test.js

```diff
@@ -58,6 +58,42 @@ test("expect CatalogItemProducts sorted by minPrice from highest to lowest when
   expect(result).toEqual(mockSortedByPriceHigh2LowCatalogItemsResponse);
 });
 
+// expect CatalogItems with offset to skip items
+test("expect CatalogitemProducts with offset to skip items", async () => {
+  let result;
+  try {
+    result = await query({
+      shopIds: [opaqueShopId],
+      offset: 1,
+      sortBy: "minPrice",
+      sortByPriceCurrencyCode: "USD"
+    });
+  } catch (error) {
+    expect(error).toBeUndefined();
+    return;
+  }
+
+  expect(result.catalogItems.nodes[0].product._id).toEqual(opaqueCatalogProductIds[1]);
+});
```

Comment:
There should be tests verifying that `hasNextPage` and `hasPreviousPage` are correct in various simple and complex cases when using offset. There are such tests in the tags integration tests for the tags query, which could be easily adapted for `catalogItems`.

Reply:
These have been added in 5c904ec9c. I don't have existing mocks for the larger product sets, so may require some assistance on the larger product mock results'